### PR TITLE
Code splitting

### PR DESCRIPTION
**Pull request by MatteoWebDesigner**
fai il bravu né 🐗

### CODE REVIEW

File: src/components/app.js [hunk 1]
```diff
@@ -1,9 +1,12 @@
 import {h, Component} from 'preact';
 import {Router} from 'preact-router';
 
+import NotFound from './notFound';
 import Header from './header';
 import Home from './home';
 import Profile from './profile';
+import Loading from './ui/loading';
```
Comment (sangallimarco) [inline src/components/app.js]: we can use the index.js inside ui folder to export components and then load them here like: 
`import {Loading} from './ui'`

File: src/components/ui/loading/index.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import {h, Component} from 'preact';
+import If from '../../utils/if';
```
Comment (sangallimarco) [inline src/components/ui/loading/index.js]: same here, we can create an index.js where we can import utils components

File: src/components/ui/button/index.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import {h, Component} from 'preact';
-import StreamComponent from '../../../lib/stream-component';
+import {StreamComponent} from '../../../libs';
```
Comment (sangallimarco) [inline src/components/ui/button/index.js]: this is after creating a barrel file

File: src/components/profile/index.js [hunk 4]
```diff
@@ -1,13 +1,10 @@
 import {h, Component} from 'preact';
 import style from './style';
 import {Streams} from './streams';
-import StreamManager from '../../lib/stream-manager';
-// ui
+import {http, StreamManager} from '../../libs';
```
Comment (sangallimarco) [inline src/components/profile/index.js]: from index.js

File: src/components/ui/loading/index.js (hunk 2 above)
Comment (MatteoWebDesigner) [inline src/components/ui/loading/index.js]: basically like preact is doing, ok.

**Pull request merged**